.ruff.toml

```diff
@@ -1,3 +1,4 @@
+fix = true
```

Comment:
I do find Ruff's auto-fixes useful, but I want to be explicit when I choose to apply them. 
```suggestion
```

Reply:
This automates applying them, but we still have to make an explicit decision to `git add` them or change them.

I prefer to autofix than the other option, which is for pre-commit to fail, and then I need to manually make edits by switching back and forth between the terminal and editor.

Ruff is an autoformatter, let's have it autoformat :)